microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -271,9 +271,7 @@ To learn more, see [Measure Performance With The RAIL Model](https://web.dev/rai
 
 To get more comfortable with the **Performance** tool, practice profiling your pages and analyzing the results.
 
-If you have any questions about your results, click the **Send Feedback** (![Send Feedback icon](./index-images/send-feedback-icon.png)) button on the main toolbar (or press **Alt+Shift+I** on Windows and Linux or **Option+Shift+I** on macOS):
-
-![The **Send Feedback** icon in Microsoft Edge DevTools](./index-images/feedback-icon.png)
+If you have any questions about your results, in the **Activity Bar**, select **Help** (![the Help icon in the Activity Bar](./index-images/help-icon.png)) > **Feedback** (or press **Alt+Shift+I** on Windows and Linux or **Option+Shift+I** on macOS).
```

Comment:
```suggestion
If you have any questions about your results, in the **Activity Bar**, select **Help** (![the Help icon in the Activity Bar](./index-images/help-icon.png)) > **Feedback**.  Or, press **Alt+Shift+I** (Windows, Linux) or **Option+Shift+I** (macOS).
```
* Perhaps no lead-in colon prior to png in this case.
* Or, you could do this seq. of the 3 elements:
```
... Help > Feedback:
.png
Or, press [key combo].
```